### DISCUSSION TITLE
IPAM allows duplicate entries

### DISCUSSION THREAD

**Discussion by travisjohnsonga** [netbox-community/netbox#13265] [Q&A] [answered]
When creating prefixes via API, I am able to create duplicate entries.  Really these should be unique per site/tenant/vrf.

I know there are use cases for having the same prefix across multiple sites, like guest wireless, but they should be unique per site like devices are.

**Comment by candlerb** [comment, marked as answer]
Netbox version?

Have you set

```
ENFORCE_GLOBAL_UNIQUE = True
```

in configuration.py?

Documentation reference: https://docs.netbox.dev/en/stable/configuration/miscellaneous/#enforce_global_unique

**Reply by travisjohnsonga** [reply]
I did not - but I shall, thank you candler!  You're a pretty helpful person!

**Reply by travisjohnsonga** [reply]
Is there a version dependency for this?

There is nothing in the settings.py for dynamic settings or the enforce global unique.

**Reply by travisjohnsonga** [reply]
I just added a section, and it worked.  Thanks again for the info

**Reply by candlerb** [reply]
Dynamic settings are configured from the web UI: (username) > Admin > Config revisions
